config/default.yml

```
@@ -610,6 +610,13 @@ Rails/SkipsModelValidations:
     - upsert_all
   AllowedMethods: []
 
+Rails/SquishedHeredocs:
+  Description: 'Checks the usage of `.squish` on SQL heredocs.'
+  Reference: 'https://rails.rubystyle.guide/#squished-heredocs'
+  Enabled: true
```

New cops should now be pending:

https://docs.rubocop.org/rubocop/versioning.html#pending-cops

```suggestion
  Enabled: 'pending'
```